tests/test_poster.py

```diff
@@ -0,0 +1,26 @@
+from tests.poster_mock import free_port, start_server
+from timeless.poster.api import Poster
+
+
+class TestPoster:
```

Comment:
@bkuzmic why did you decide use class based test? I think it's better to make a fixture that will start server. Also it will be easier to pass fixtures to tests, without using a specific decorator made for passing fixtures to class methods. wdyt?

Reply:
@vryazanov Just reverted the old code. I'll refactor it.